src/map.rs

```diff
@@ -2416,9 +2416,8 @@ where
 {
     type Value = HashMap<K, V, S>;
 
-    // TODO: Create an error message
-    fn expecting(&self, _f: &mut Formatter<'_>) -> fmt::Result {
-        todo!("Create an error message");
+    fn expecting(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        write!(f, "a map could no be deserialized");
```

Comment:
I think this should just be "a map". See [`hashbrown`](https://github.com/rust-lang/hashbrown/blob/17572d45dbe31fb1a932c04fe56e9abf777ee4ea/src/external_trait_impls/serde.rs#L62) and the [serde documentation](https://docs.rs/serde/1.0.105/serde/de/trait.Visitor.html#tymethod.expecting):
> The message should complete the sentence "This Visitor expects to receive ...", for example the message could be "an integer between 0 and 64". The message should not be capitalized and should not end with a period.